README.md

```diff
@@ -461,3 +467,12 @@ Where the suffix after `knn` or `eps` specifies $k$ (number of neighbours) or $\
 | Name      | Description   | Dimensionality |
 | ----------- | ----------- | ----------- |
 | `edge_vectors` | Edge directional vectors (unit-normalized)        |      1  |
+
+# For developers
+To keep with the code style for the proteinworkshop repository, please format your commits requests before opening a merge request with the following lines:
```

Comment:
We should probably use `proteinworkshop` here instead of proteinworkshop to stay consistent with the rest of the README's naming scheme for the repo.

Reply:
done